app/data_utils.py

```diff
@@ -406,9 +406,9 @@ def pivot_and_clean_dates(coh_long, coh_wide):
         index='delta_days',
         columns='second_date',
         values='first_date')
-    date_wide = date_wide.applymap(lambda x: pd.to_datetime(x)
-                                   .strftime('%b %d, %Y') if x is not pd.NaT
-                                   else x)
+    date_wide = date_wide.map(lambda x: pd.to_datetime(x)
```

Comment:
App crashes for me with `AttributeError: 'DataFrame' object has no attribute 'map'`. Seems to be being caused by this line

Reply:
This was to address a future warning error. datafram.map was added in pandas v2.1. requirements.txt updated

Reply:
https://github.com/Volcano-Risk-Reduction-in-Canada/Volcanic-Interpretation-Workbench/pull/68/commits/a81ad24a6d97c4b153b9be848b8817f0380caed9